### PR TITLE
{vis}[GCCcore/8.2.0] JasPer v2.0.14

### PR DESCRIPTION
**Pull request by akesandgren**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/j/JasPer/JasPer-2.0.14-GCCcore-8.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,35 @@
+easyblock = 'CMakeMake'
+
+name = 'JasPer'
+version = '2.0.14'
+
+homepage = 'http://www.ece.uvic.ca/~frodo/jasper/'
```
Comment (boegel) [inline easybuild/easyconfigs/j/JasPer/JasPer-2.0.14-GCCcore-8.2.0.eb]: @akesandgren Please change to `https://`

File: easybuild/easyconfigs/j/JasPer/JasPer-2.0.14-GCCcore-8.2.0.eb [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+description = """
+ The JasPer Project is an open-source initiative to provide a free
+ software-based reference implementation of the codec specified in
+ the JPEG-2000 Part-1 standard.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '8.2.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['http://www.ece.uvic.ca/~frodo/jasper/software/']
```
Comment (boegel) [inline easybuild/easyconfigs/j/JasPer/JasPer-2.0.14-GCCcore-8.2.0.eb]: @akesandgren Please change to `https://`

File: easybuild/easyconfigs/j/JasPer/JasPer-2.0.14-GCCcore-8.2.0.eb (hunk 1 above)
Comment (akesandgren) [reply easybuild/easyconfigs/j/JasPer/JasPer-2.0.14-GCCcore-8.2.0.eb]: done

File: easybuild/easyconfigs/j/JasPer/JasPer-2.0.14-GCCcore-8.2.0.eb (hunk 2 above)
Comment (akesandgren) [reply easybuild/easyconfigs/j/JasPer/JasPer-2.0.14-GCCcore-8.2.0.eb]: done

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 5 out of 5 (1 easyconfigs in this PR)
b-an03.hpc2n.umu.se - Linux ubuntu 16.04, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 2.7.12
See https://gist.github.com/c969475692553519c6e67fb92a492615 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2402.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/cbcbf1fc1a019f9e59827b830873f2cb for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2077.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/e06a9610deaa9a5a803ca9e0e626c337 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (1 easyconfigs in this PR)
node3134.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/171d547214cc4e6950eaaf691cfd75c6 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @akesandgren!

**Pull request merged**